vtex/admin-promotions
# Fix time range validation

#### What is the purpose of this pull request?
Fix validation of time range select on recurring promotions.
Note: the recurrence config is not being saved correctly yet. This PR is only about fixing the validation problem.  

#### What problem is this solving?
When there is an error on those time range selects, the application breaks completely.

#### How should this be manually tested?
Try creating or editing a promotion. Open the folded "Recurrency settings" menu and select the 
"Specific times of the day" option. Input an invalid range (end before start; leave a field empty) and save the promotion. It is expected to focus on the invalid field and show the validation message.
https://vlauxexp--pricingqa.myvtex.com/admin/app/promotions/1c3ab5a4-533c-4b04-904e-c5719f8fb755/

#### Types of changes
